From 5ce9cf12fedf6d2b84ed03e7b4a0c403a5a89cd1 Mon Sep 17 00:00:00 2001
From: cnauroth <cnauroth@apache.org>
Date: Wed, 12 Nov 2014 21:54:06 -0800
Subject: [PATCH 662/791] HADOOP-11291. Log the cause of SASL connection
 failures. Contributed by Stephen Chu.

(cherry picked from commit 7dae5b5a880205fd0125c143c44498db79c19392)
---
 .../main/java/org/apache/hadoop/ipc/Client.java    |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java
index be0fd40..53c73ea 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java
@@ -667,7 +667,7 @@ public Object run() throws IOException, InterruptedException {
               String msg = "Couldn't setup connection for "
                   + UserGroupInformation.getLoginUser().getUserName() + " to "
                   + remoteId;
-              LOG.warn(msg);
+              LOG.warn(msg, ex);
               throw (IOException) new IOException(msg).initCause(ex);
             }
           } else {
-- 
1.7.9.5

